### PR TITLE
fix(VerticalNavigation): truncate text on slider navigation

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: packages/core/src/components/VerticalNavigation/NavigationSlider/NavigationSlider.styles.tsx [hunk 1]
```diff
@@ -27,3 +33,12 @@ export const StyledListItem = styled(HvListItem)({
     },
   },
 });
+
+export const StyledOverflowTooltip = styled(
+  (props: HvOverflowTooltipProps) => <HvOverflowTooltip {...props} />,
+  transientOptions
+)(({ $hasEndAdornment }: { $hasEndAdornment: boolean }) => ({
+  ...(!$hasEndAdornment && {
+    paddingRight: 8,
```
Comment (zettca) [inline packages/core/src/components/VerticalNavigation/NavigationSlider/NavigationSlider.styles.tsx:42]:
```suggestion
    paddingRight: theme.space.xs,
```

File: packages/core/src/components/VerticalNavigation/VerticalNavigation.test.tsx [hunk 2]
```diff
@@ -208,6 +208,23 @@ const SliderSample = () => {
...
+      observe: vi.fn(),
+      unobserve: vi.fn(),
+      disconnect: vi.fn(),
+    }));
+  });
+
+  afterEach(() => {
+    window.ResizeObserver = ResizeObserver;
+    vi.restoreAllMocks();
+  });
```
Comment (zettca) [inline packages/core/src/components/VerticalNavigation/VerticalNavigation.test.tsx:226]: I needed this in my PR as well.
I think we should move to the global test setup, as this is part of the DOM APIs that JSDOM currently doesn't mock

Comment (zettca) [reply packages/core/src/components/VerticalNavigation/VerticalNavigation.test.tsx:226]: I opened a PR for this https://github.com/lumada-design/hv-uikit-react/pull/3354

Comment (plagoa) [reply packages/core/src/components/VerticalNavigation/VerticalNavigation.test.tsx:226]: Nice, I'll leave this as is then.

**Review by MEsteves22 — APPROVED** [review]

**Pull request merged**